src/paging.rs

```diff
@@ -37,6 +37,17 @@ pub enum VaRange {
     Upper,
 }
 
+/// Which exception level a page table is for.
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub enum ExceptionLevel {
```

Comment:
This should be TranslationRegime, not ExceptionLevel: in addition to EL2 vs EL2&0, there is also stage-2 and stage-3 translation that needs to be taken into account.

Reply:
Done. I've kept it to just stage 1 non-secure for now (except for EL3).